### ISSUE TITLE
Upgrade node-webkit

### ISSUE THREAD

**Issue by cldwalker** [LightTable/LightTable#1701]
I believed this is blocked by css variables. I'm hoping we'll upgrade to a non-forked version of node-webkit so our scripts can eventually build without dependencies on privately-held tarballs - see #1666. If there's a script for building that tarball, it would be great to put it in the repo.

**Comment by cldwalker** [comment]
Closing since this is no longer relevant
